### ISSUE TITLE
on Hyper v 3.0.0 looks weird

### ISSUE THREAD

**Issue by IgorMuzyka** [hummal/hyper-monokai-deluxe#3]
<img width="780" alt="Screen Shot 2019-05-07 at 00 07 08" src="https://user-images.githubusercontent.com/4402333/57255310-1e599400-705c-11e9-8d77-8294d6a90596.png">

**Comment by philipp-winterle** [comment]
Hey Igor,

I checked the latest version (3.0.2) of hyper with hyper-monokai-deluxe and it looks like it should.
Maybe some other plugins not working as expected? 
What is your config?

![hyper](https://user-images.githubusercontent.com/633711/57515526-2727bf80-7313-11e9-9d14-6a507464e14f.png)

**Comment by IgorMuzyka** [comment]
here is a list of my hyper plugins 
```js
    "hyperminimal",
    "hyperterm-cursor",
    "hyperterm-paste",
    "hyper-search",
    "hyper-monokai-deluxe"
```

**Comment by travVT** [comment]
I'm getting the same thing with Hyper 3.0.2 on my Mac. If I uninstall and reinstall the plugin it looks okay until the next time I open Hyper. Any ideas? Thanks! :)

**Comment by philipp-winterle** [comment]
I was able to reproduce it now. Seems to have something to do with OSX strategy of displaying motion blur. Will have a closer look if I can do something or if its a hyper.js issue

**Comment by philipp-winterle** [comment]
Fixed in 1.0.5

Try the new option "theme" with value "reloaded" for a better css layout

Thank your for reporting

**Comment by travVT** [comment]
Thanks @hummal, it's working great now! :)
